python2/raygun4py/raygunprovider.py

```diff
@@ -31,8 +31,9 @@ class RaygunSender:
 
     api_key = None
     endpointprotocol = 'https://'
-    endpointhost = 'api.raygun.io'
```

Comment:
Do we also  need to include release notes so people can change firewall rules?

Reply:
Good thought, I might pull this change out as a separate PR and we can discuss this as a more general update across providers.

Reply:
Have reverted this change, will address in a later PR

Reply:
Have reverted this change, will address in a later PR